pom.xml

```diff
@@ -18,5 +18,23 @@
         </dependency>
 
     </dependencies>
+
+<build>
```

Comment:
`<build>...</build>` should be indented. `<plugins>...</plugins>` should be outdented.